ssl/statem/extensions_clnt.c

```diff
@@ -806,10 +810,11 @@ EXT_RETURN tls_construct_ctos_early_data(SSL_CONNECTION *s, WPACKET *pkt,
             }
 
             psksess = SSL_SESSION_new();
+            const int version = SSL_CONNECTION_IS_DTLS(s) ? DTLS1_3_VERSION : TLS1_3_VERSION;
```

Comment:
Style nit: declarations at the start of a block. Also put a blank line after declarations.